internal/command/migrate_create.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 Sergey Novichkov. All rights reserved.
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+package command
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gozix/di"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+// NewMigrateCreate is subcommand constructor.
+func NewMigrateCreate(ctn di.Container) *cobra.Command {
+	return &cobra.Command{
+		Use:   "create",
+		Short: "Create named migration",
+		Long: `Normally each migration is run within a transaction. 
+	However some SQL commands (for example creating an index concurrently in PostgreSQL) 
+	cannot be executed inside a transaction. In order to execute such a command in a migration, 
+	the migration can be run using the notransaction option https://github.com/rubenv/sql-migrate`,
+		Args: cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return ctn.Call(func(ctx context.Context, logger *zap.Logger) (err error) {
+				const extension string = "sql"
```

Comment:
Make please this const as package global const.

Reply:
Moved to global scope